src/messageReactionAdd.ts

```diff
@@ -1,45 +1,71 @@
 import { MessageReaction } from 'discord.js';
 
 import config from './config';
+import { log } from './logger';
 
 const { REACTIONS_COUNT, ROLE } = config;
 
 const getTotalReactionCount = async (reaction: MessageReaction) => {
     // Map through each reaction emoji and return the count
-    const getReactionCounts = reaction.message.reactions.cache.map(async (value) => {
-        if (value.partial) {
-            try {
-                await value.fetch();
-            } catch (error) {
-                return Promise.resolve(0)
+    const getReactionCounts = reaction.message.reactions.cache
+        .filter((r) => !r.users.cache.has(reaction.message.author.id)) // remove the message author's reactions
```

Comment:
I'm not sure if this is correct or not😁 : What if more than one user (including the author) reacted with the same reaction, this line will discard the reacted users because the author reacted with the same reaction.

Reply:
Hmm 🤔 . You are probably right. But on the flip side, the author should not really give their own message a reaction 😂

Reply:
There is nothing say they can't 😂